client/asset/btc/electrum_client_test.go

```diff
@@ -0,0 +1,156 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+//go:build electrumlive
+
+package btc
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"sync"
+	"testing"
+	"time"
+
+	"decred.org/dcrdex/client/asset/btc/electrum"
+	"decred.org/dcrdex/dex"
+	dexltc "decred.org/dcrdex/dex/networks/ltc"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/txscript"
+)
+
+func Test_electrumWallet(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	const walletPass = "walletpass" // set me
+	ewc := electrum.NewWalletClient("user", "pass", "http://127.0.0.1:5678")
```

Comment:
Should this port should be `16789` now?

---
When using on simnet, this test is failing with `no headers retrieved`.

Reply:
Ah, this little playground of a test is targeting testnet (`params: dexltc.TestNet4Params` below) because the txids are historical and you can investigate on the block explorer.  Although now that I revisit this, I had done this test on the "bad" non-mweb fork so I should regenerate the txids and and explorer links